src/main/java/at/esque/kafka/topics/KafkaMessage.java

```diff
@@ -17,11 +20,14 @@
 import org.apache.kafka.common.header.Header;
 
 public class KafkaMessage {
-
     private LongProperty offset = new SimpleLongProperty();
+    @CsvCustomBindByName(column = "partition", converter = IntegerPropertyConverter.class)
```

Comment:
Generell: Wie schon in unserer Besprechung erwähnt, gefällt mir Variante 1 besser, weil dort der spezielle Use-Case "MessageBooks" schön von UI-Aspekten entkoppelt wird. Sollte sich jemals etwas an der UI-Klasse ändern, sollte mit Variante 1 das MessageBooks-Feature davon unabhängig weiterhin funktionieren, ohne dort etwas nachziehen zu müssen.

Reply:
Okay, danke für deine Reviews!
Ich würde Variante 1 auch bevorzugen aufgrund der besseren Trennung für die 2 unterschiedlichen Use-cases.